wrappers/swift/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.2)
+
+PROJECT(Swift_Wrapper)
+
+message("${CMAKE_CURRENT_SOURCE_DIR}")
+
+enable_language(Swift)
+
+# TODO: add logic to build dependencies
```

Comment:
Since, this is camke file only for the swift wrappers code and it has dependecy on objc, it would require objc built lib to be linked in this make files to generate the proper proj files.
I have added the dependencies and they seem to be working. I would be raising those in PR once this one is in.